### PR TITLE
A question issue template added

### PR DESCRIPTION
**Pull request by WhiteBlackGoose**
There are many projects which sadly discourage asking questions as easy as opening issues. There might be reasons for it, but the community of `interactive` is very civil and small.

So in my opinion, it might be a good idea to actively encourage asking questions. What do you think?

### CODE REVIEW

**Comment by jonsequitur** [comment]
What about enabling GitHub Discussions?

Thoughts, @LadyNaggaga?

**Comment by WhiteBlackGoose** [comment]
Oh, discussions is also a great idea. Although I'd make both (a question template AND dicussions).

**Comment by LadyNaggaga** [comment]
I love the idea of fostering a community where we can ask questions.  I like the idea of creating question issue that we share on GitHub discussions and discord.  That way we can encourage the community to support questions and learners more

**Comment by LadyNaggaga** [comment]
I love the idea of fostering a community where we can ask questions.  I like the idea of creating question issue that we share on GitHub discussions and discord.  That way we can encourage our communities to participate

File: .github/ISSUE_TEMPLATE/question.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+labels: question
+assignees: ''
+
+---
+
+**The package and version I'm asking about**: 
+<!--
+There are a few of ways to find this out:
+* In a notebook, run the `#!about` magic command. 
+* At the command line, run `dotnet interactive --version`.
```
Comment (jonsequitur) [inline .github/ISSUE_TEMPLATE/question.md:14]:
```suggestion
* At the command line, run `dotnet interactive --version`. (_Non-VS Code only._)
```

File: .github/ISSUE_TEMPLATE/question.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+-->
+
+### Question
+<!--Ask your question here. Be specific-->
+
+<!--You can also ask your questions on official .NET server: https://discord.gg/3pvut9YujN -->
```
Comment (jonsequitur) [inline .github/ISSUE_TEMPLATE/question.md:20]:
```suggestion
<!--You can also ask your questions on the .NET Discord server: https://discord.gg/3pvut9YujN -->
```

**Review by jonsequitur — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/question.md (hunk 1 above)
Comment (WhiteBlackGoose) [reply .github/ISSUE_TEMPLATE/question.md:14]: @jonsequitur someone who is going to ask a question will see a plain text, not a formatted one. So we probably don't need those `_`

**Comment by WhiteBlackGoose** [comment]
@jonsequitur review now please

File: .github/ISSUE_TEMPLATE/question.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ assignees: ''
 <!--
 There are a few of ways to find this out:
 * In a notebook, run the `#!about` magic command. 
-* At the command line, run `dotnet interactive --version`.
+* At the command line, run `dotnet interactive --version`. Non-VS Code only.
```
Comment (jonsequitur) [inline .github/ISSUE_TEMPLATE/question.md:14]:
```suggestion
* At the command line, run `dotnet interactive --version`. (Non-VS Code only.)
```

**Comment by WhiteBlackGoose** [comment]
@jonsequitur check now

**Review by jonsequitur — APPROVED** [review]

**Pull request merged**